apollo-compiler@0.7.1 (#503)
apollographql/apollo-rs: crates/apollo-compiler/CHANGELOG.md

## Documentation -->

# [0.7.1](https://crates.io/crates/apollo-compiler/0.7.1) - 2023-03-28

## Features

> - validate indirectly self-referential directives by [goto_bus_stop] in [pull/494]

Reviewer: ```suggestion
- validate indirectly self-referential directives by [goto-bus-stop] in [pull/494]
```

Author: why do i keep doing this!! ahhh

## Fixes

> - include built-in enum types in db.type_system() query by [Simon Sapin] in [pull/501]

Reviewer: ```suggestion
- include built-in enum types in `db.type_system()` query by [SimonSapin] in [pull/501]
```

> - field.field_definition() works for interface types by [zackangelo] in [pull/502]

Reviewer: ```suggestion
- `field.field_definition()` works for interface types by [zackangelo] in [pull/502]
```
